src/components/SharedComponents/ObservationViews/EmptyList.js

```diff
@@ -11,8 +12,7 @@ const EmptyList = ( ): Node => {
 
   return (
     <View style={viewStyles.center}>
-      <Text style={textStyles.text} testID="ObsList.emptyList">welcome to inaturalist!</Text>
-      <Text style={textStyles.text}>make sure you're logged in to fetch observations</Text>
+      <Text style={textStyles.text} testID="ObsList.emptyList">{t( "iNaturalist-is-a-community-of-naturalists." )}</Text>
```

Comment:
This causes a crash when signed out, should be `iNaturalist-is-a-community-of-naturalists` (no period)